<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>symmetry_rotation</class>
 <widget class="qMRMLWidget" name="symmetry_rotation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>572</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>Inputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="volumeInputLabel">
        <property name="text">
         <string>Volume Input</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="volumeInputqMRMLNodeComboBox">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="inputSegmentationLabel">
        <property name="text">
         <string>Input Segmentation</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="qMRMLNodeComboBox" name="inputSegmentationqMRMLNodeComboBox">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLSegmentationNode</string>
         </stringlist>
        </property>
        <property name="noneEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Limit Which Segmentations to run calculations on</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QTextEdit" name="textEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="outputsCollapsibleButton">
     <property name="text">
      <string>Outputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0" colspan="2">
       <widget class="ctkPathLineEdit" name="PathLineEdit">
        <property name="contextMenuPolicy">
         <enum>Qt::ActionsContextMenu</enum>
        </property>
        <property name="label">
         <string/>
        </property>
        <property name="filters">
         <set>ctkPathLineEdit::Dirs|ctkPathLineEdit::Files|ctkPathLineEdit::Readable</set>
        </property>
        <property name="showHistoryButton">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>Append Data</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="advancedCollapsibleButton">
     <property name="text">
      <string>Advanced</string>
     </property>
     <property name="collapsed">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Invert threshold: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="invertOutputCheckBox">
        <property name="toolTip">
         <string>If checked, values above threshold are set to 0. If unchecked, values below are set to 0.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="applyButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>symmetry_rotation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>volumeInputqMRMLNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>symmetry_rotation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputSegmentationqMRMLNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>symmetry_rotation</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>checkBox_2</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
